releases/R2023b/scripts/initAdminVM.sh

```diff
@@ -1,3 +1,4 @@
+<<<<<<< Updated upstream
```

Comment:
Looks like there are errors in this file when you may have tried to resolved merge conflicts. You shouldn't have to commit this file.